src/renderer/components/about-app.tsx

```diff
@@ -175,6 +184,13 @@ export default class AboutApp extends React.Component<{}, IState> {
     }
   }
 
+  /**
+   * Confirm to close modal
+   */
+  public confirm(): void {
+    window.close();
```

Comment:
I think we need to close the window from the main process

Reply:
Agree, @NguyenTranHoangSym have a look at the way "copy" function is implemented. This API allows IPC

Reply:
the dialog itself has its own window life cycle. I dont seem to find a method trigger destroying a modal from BrowserWindow. What I can possibly do is to listen to the destroy event and do certain thing, like gain back focus

Reply:
I see, let me see how should it be done, its not usual to pass callback.

Reply:
Basically:
- we send an event to the main process on a dedicated channel (ipcRenderer.send)
- main process is listening to a channel (ipcMain)
- once we receive the event, we perform the action (here, we close the window and destroy its reference in window-handler.ts)